modules/openthread/Kconfig.features

```diff
@@ -92,6 +92,12 @@ config OPENTHREAD_CSL_RECEIVER
 	help
 	  Enable CSL Receiver support for Thread 1.2
 
+config OPENTHREAD_DEVICE_PROP_LEADER_WEIGHT
+	bool "Device props for leader weight"
+	help
+	  Enable the device properties which are then used to determine and set
+	  the Leader Weight.
+
```

Comment:
I'm wondering if we should add some dependency with default regarding OT version here, as in OT, the flag is enabled by default for version 1.3.1 and later: https://github.com/nrfconnect/sdk-openthread/blob/79a3a26db601571829d803b61d6b390a87ea9810/src/core/config/mle.h#L101
Thoughts?

Reply:
@maciejbaczmanski @edmont Is there a way to do a `default y if THREAD_V131` or similar? I seem to remember we had a variable like this, just don't remember the exact syntax.

Reply:
@maciejbaczmanski @edmont Fixed (just inversed the logic to avoid the "not" operation) - is this ok?